src/java.base/share/classes/sun/nio/ch/ChannelInputStream.java

```diff
@@ -46,7 +46,7 @@
  * @author Mark Reinhold
  */
 class ChannelInputStream extends InputStream {
-    private static final int DEFAULT_BUFFER_SIZE = 8192;
+    static final int DEFAULT_BUFFER_SIZE = 8192;
```

Comment:
I don't think COS should be using CIS.DEFAULT_BUFFER_SIZE, maybe it should be moved to Streams which is the factory class for the streams based on channels.

Reply:
Moved to `Streams` in a9af89cff869c441c6dd04f635a4d93e7757f2fa.